.pre-commit-config.yaml

```diff
@@ -16,7 +16,6 @@ repos:
   rev: v4.3.0
   hooks:
   - id: check-case-conflict
-  - id: check-executables-have-shebangs
```

Comment:
all of our `*.sh` files in the `install/` directory don't have shebangs, let's get rid of this

Reply:
They're also not executable.

```
/Users/chadwhitacre/workbench/getsentry/self-hosted/install
total 88
-rw-r--r--  1 chadwhitacre  staff  1666 Mar 17 15:04 _lib.sh
-rw-r--r--  1 chadwhitacre  staff   208 Mar 17 15:04 _min-requirements.sh
-rw-r--r--  1 chadwhitacre  staff   164 Mar 17 15:04 bootstrap-snuba.sh
-rw-r--r--  1 chadwhitacre  staff   446 Mar 17 15:04 build-docker-images.sh
-rw-r--r--  1 chadwhitacre  staff   642 Mar 17 15:04 check-latest-commit.sh
-rw-r--r--  1 chadwhitacre  staff  2937 Mar 17 15:04 check-minimum-requirements.sh
-rw-r--r--  1 chadwhitacre  staff   520 Mar 17 15:04 create-docker-volumes.sh
-rw-r--r--  1 chadwhitacre  staff   694 Mar 17 15:04 create-kafka-topics.sh
-rw-r--r--  1 chadwhitacre  staff   716 Mar 17 15:04 dc-detect-version.sh
-rw-r--r--  1 chadwhitacre  staff  1151 Mar 17 15:04 detect-platform.sh
-rw-r--r--  1 chadwhitacre  staff   211 Mar 17 15:04 ensure-files-from-examples.sh
-rw-r--r--  1 chadwhitacre  staff  1684 Mar 17 15:04 ensure-relay-credentials.sh
-rw-r--r--  1 chadwhitacre  staff  7610 Mar 17 15:04 error-handling.sh
-rw-r--r--  1 chadwhitacre  staff   641 Mar 17 15:04 generate-secret-key.sh
-rw-r--r--  1 chadwhitacre  staff   964 Mar 17 15:04 geoip.sh
-rw-r--r--  1 chadwhitacre  staff  1531 Mar 17 15:04 install-wal2json.sh
-rw-r--r--  1 chadwhitacre  staff  2719 Mar 17 15:04 parse-cli.sh
-rw-r--r--  1 chadwhitacre  staff   383 Mar 17 15:04 set-up-and-migrate-database.sh
-rw-r--r--  1 chadwhitacre  staff   352 Mar 17 15:04 turn-things-off.sh
-rw-r--r--  1 chadwhitacre  staff   608 Mar 17 15:04 update-docker-images.sh
-rw-r--r--  1 chadwhitacre  staff   972 Mar 17 15:04 wrap-up.sh
```

Reply:
🤦‍♂️